tft.py

```diff
@@ -256,6 +256,9 @@ def queue() -> None:  # pylint: disable=too-many-branches
 
         if not LCU_INTEGRATION.create_lobby():
             time.sleep(5)
+        else:

```

Comment:
Not that it matters, but whenever I see `else` statements I see if I can avoid them ;)
`if not create_lobby(): time.sleep(3)` and then a `time.sleep(2)` would do the trick, but again, I don't care enough to not approve :D

Reply:
No complaints here, I'd argue it's a little cleaner, sorta, and honestly I don't have a preference right now 🤷‍♂️